lib/error.js

```diff
@@ -4,14 +4,12 @@ var util = require('util'),
     Controller = require('hmpo-form-controller');
 
 function getArgs(type, args) {
-    if (type === 'maxlength') {
-        return { maxlength: args[0] };
-    } else if (type === 'minlength') {
-        return { minlength: args[0] };
-    } else if (type === 'exactlength') {
-        return { exactlength: args[0] };
-    } else if (type === 'past') {
+    if (type === 'past') {
         return { age: args.join(' ') };
+    } else if (_.isArray(args) && typeof type === 'string') {
+        var obj = {};
+        obj[type] = args[0];
+        return obj;
     } else {
         return {};
```

Comment:
This has nothing to do with this particular PR. But in the future it might be worth asking whether this is the correct behaviour. Is it not just silent failure? Wouldn't it be better to throw an exception here?